config/pillar/top.sls

```diff
@@ -0,0 +1,3 @@
+base:
+  '*'
+    - hana.hana
```

Comment:
Is this correct? This forces the user to have a fixed `hana.sls` pillar file to the user.

Reply:
Yes I think is good to force to have 1 file as global conf imho. Otherwise I could not spot any valid alternatives